app/controllers/authenticate_controller.rb

```diff
@@ -87,6 +87,17 @@ def authenticate_oidc
     authenticate(input)
   end
 
+  def authenticate_gcp
+    params[:authenticator] = "authn-gcp"
```

Comment:
Isn't a single quote is preferred in ruby?

Reply:
```
The difference between single and double quoted strings in Ruby is the way the string definitions represent escape sequences.

In double quoted strings, you can write escape sequences and Ruby will output their translated meaning. A \n becomes a newline.
In single quoted strings however, escape sequences are escaped and return their literal definition. A \n remains a \n.
Both string types have their use cases of course. It’s likely you’ll use double quoted strings with escape sequences to use their translated meaning, a newline or a tab.

Single quotes are useful for avoiding escaping escape sequences themselves. Useful for demonstrating their usage or avoiding accidentally using escape sequences you didn’t intent to.
```

in this case i don't see a preference. do you think we should have some alignment? if so - why single quote? can you share a link?

Reply:
@jonahx do you have an opinion on this?